pulser-simulation/pulser_simulation/simconfig.py

```diff
@@ -54,6 +56,11 @@ class SimConfig:
             noise types:
 
             - "dephasing": Random phase (Z) flip
+            - "depolarizing": model of decohering where the qubit undergoes a
+              bit-flip error, phase-flip error or both errors.
+            - "gen_noise": general model of noise channels where the qubit
+            undergoes a modification according to the specified noise
+            operators, it encapsulates dephasing and depolarizing channels.
```

Comment:
From this description, it's not clear to me: what's the difference between having `gen_noise` vs `dephasing`+ `depolarizing`?

Reply:
The difference is that you can design any operator noises.
For example: dephasing and depolarizing channels are instances of the general noise channel